source/quickinstallationguide/qig.rst

```diff
@@ -71,75 +71,107 @@ CloudStack. We will go over the steps to prepare now.
+that there will be no DHCP server in this environment we will be manually 
+configuring your network interface. 
 
-Unfortunately, this configuration will not permit you to connect to the 
-network, and is also unsuitable for our purposes with CloudStack. We want to 
-configure that file so that it specifies the IP address, netmask, etc., as 
-shown in the following example:
+Connecting via the console you should login as root. We will start by creating
+the bridge that Cloudstack will use for networking. Create and open
+/etc/sysconfig/network-scripts/ifcfg-cloudbr0 and add the following settings:
 
 .. note:: 
-   You should not use the Hardware Address (aka the MAC address) from our 
-   example for your configuration. It is network interface specific, so you 
-   should keep the address already provided in the HWADDR directive.
-
-:: 
+   IP Addressing - Throughout this document we are assuming that you will have 
+   a /24 network for your CloudStack implementation. This can be any RFC 1918 
+   network. However, we are assuming that you will match the machine address 
+   that we are using. Thus we may use 172.16.10.2 and because you might be 
+   using the 192.168.55.0/24 network you would use 192.168.55.2
+   
+::
 
-   DEVICE=eth0
-   HWADDR=52:54:00:B9:A6:C0
-   NM_CONTROLLED=no
+   DEVICE=cloudbr0
+   TYPE=Bridge
    ONBOOT=yes
    BOOTPROTO=none
+   IPV6INIT=no
+   IPV6_AUTOCONF=no
+   DELAY=5
    IPADDR=172.16.10.2
-   NETMASK=255.255.255.0
    GATEWAY=172.16.10.1
+   NETMASK=255.255.255.0
    DNS1=8.8.8.8
    DNS2=8.8.4.4
 
-.. note:: 
-   IP Addressing - Throughout this document we are assuming that you will have 
-   a /24 network for your CloudStack implementation. This can be any RFC 1918 
-   network. However, we are assuming that you will match the machine address 
-   that we are using. Thus we may use 172.16.10.2 and because you might be 
-   using the 192.168.55.0/24 network you would use 192.168.55.2
+Save the configuration and exit. We will then edit the interface so that it
+makes use of this bridge. Enter this command to find your interfaces: 
+
+.. note::
+   CentOS 7 has implemented 'Predictable Network Interface Names<https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/>'_ and as such, 
+   the following instructions will depend on your configuration, and interface
+   names are used only for the sake of simplicity.
+
+.. parsed-literal::
+   # ls /etc/sysconfig/network-scripts | grep ifcfg
+   
+This should return three results: ifcfg-lo, ifcfg-cloudbr0, and ifcfg-enp3s0. The first being loopback and the second being the interface we've just created, open ifcfg-enp3s0. Replace it's current configuration with the following: 
+
+::
+   TYPE=Ethernet
+   PROXY_METHOD=none
+   BROWSER_ONLY=no
+   BOOTPROTO=none
+   DEFROUTE=yes
+   IPV6INIT=no
+   NAME=enp5s0
+   UUID=26f024e6-1113-416e-b319-58ebec347886
+   DEVICE=enp3s0
+   ONBOOT=yes
+   BRIDGE=cloudbr0
```

Comment:
You can simply say that, for the default ethernet device simply configure: (replace enp3s0 accordingly)
```
# Setup networking
cat > /etc/sysconfig/network-scripts/ifcfg-enp3s0 <<EOF
TYPE=Ethernet
DEVICE=eth0
ONBOOT=yes
BOOTPROTO=static
BRIDGE=cloudbr0
NM_CONTROLLED=no
EOF
```

Reply:
I'm not sure what you mean. Instead of telling the bridge to use the default ethernet device, tell the DET to use the bridge?

Reply:
I guess Rohit meant is, to not explain the grep process, predictable interface names etc - but just to replace example interface name with your own interface.

So I would also like to keep eth0 sa a simplified name, but mention to replace eth0 with your own default interface name. Rest of the bridge config section stays the same.

Sounds reasonably, just a bit of simplification of the guide.

Reply:
Sounds perfectly reasonable. Change implemented.